integration/securitypolicy_test.go

```diff
@@ -0,0 +1,178 @@
+package integration
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/TykTechnologies/tyk-operator/api/v1alpha1"
+	"github.com/matryer/is"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/e2e-framework/klient/k8s"
+	"sigs.k8s.io/e2e-framework/klient/wait"
+	"sigs.k8s.io/e2e-framework/klient/wait/conditions"
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+)
+
+func TestSecurityPolicyStatusIsUpdated(t *testing.T) {
+	is := is.New(t)
```

Comment:
Maybe we should rename `is` to `eval` or something else to prevent duplicated declarations (package name `is` and local variable `is` ).